### PR TITLE
Added: carbon stats and coming soon card on the home page

### PR DESCRIPTION
**Pull request by iosdevashok**
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-TODO)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/TODO)

### Description
TODO

### Pull requests checks where applicable
- [ ] Fill in the three TODOs above (tickets number and description of your work)
- [ ] The PR name follows our [naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Accessibility implemented and tested (minimum Dynamic Text and VoiceOver)
- [ ] Unit tests written and passing
- [ ] Documentation / comments for complex code and public methods

### CODE REVIEW

**Comment by iosdevashok** [comment]
@madtocc  Ive completed both tickets (#16 and #17 ) and pushed on the same branch. 
Simulator Screen Shot - iPhone 14 - 2023-03-17 at 13 41 41

File: Client/Assets/Images.xcassets/ic_bridge.imageset/ic_bridge.png
Comment (madtocc) [inline Client/Assets/Images.xcassets/ic_bridge.imageset/ic_bridge.png:1]: add icon without background. pdf are preferred or a bit higher. quality of icon seems not ok on phone.

File: Client/Assets/Images.xcassets/ic_stacking.imageset/ic_stacking.png
Comment (madtocc) [inline Client/Assets/Images.xcassets/ic_stacking.imageset/ic_stacking.png:1]: add icon without background. pdf are preferred or a bit higher. quality of icon seems not ok on phone.

File: Client/Assets/Images.xcassets/ic_swap.imageset/ic_swap.png
Comment (madtocc) [inline Client/Assets/Images.xcassets/ic_swap.imageset/ic_swap.png:1]: add icon without background. pdf are preferred or a bit higher. quality of icon seems not ok on phone.

File: Client/Assets/Images.xcassets/ic_wallet.imageset/ic_wallet.png
Comment (madtocc) [inline Client/Assets/Images.xcassets/ic_wallet.imageset/ic_wallet.png:1]: add icon without background. pdf are preferred or a bit higher. quality of icon seems not ok on phone.

File: Client/Frontend/Home/LogoHeader/HomeLogoHeaderCell.swift
Comment (madtocc) [inline Client/Frontend/Home/LogoHeader/HomeLogoHeaderCell.swift:1]: add a view with gradient background + cornerRadius instead of just the image

**Review by madtocc — CHANGES_REQUESTED** [review]

**Comment by iosdevashok** [comment]
Simulator Screen Shot - iPhone 14 - 2023-03-22 at 09 35 51

**Review by madtocc — APPROVED** [review]

**Pull request merged**